### DISCUSSION TITLE
SSL Error

### DISCUSSION THREAD

**Discussion by LettuceJuice** [ORelio/Minecraft-Console-Client#1781] [Q&A] [answered]
When attempting to connect to minecraft.net to login, I get "Minecraft Login failed :SSL Error.'"
I'm not sure what's causing this or how to fix, anyone know?

**Comment by ReinforceZwei** [comment, marked as answer]
See #1780

**Reply by LettuceJuice** [reply]
Thank you

**Reply by ORelio** [reply]
Marked as answered since the issue is fixed in #1780
